test/integration/test_container_resolvers.py

```diff
@@ -144,15 +155,19 @@ def handle_galaxy_config_kwds(cls, config) -> None:
 
     def _clear_container_cache(self):
         """
-        clear all possibe container caches (ie docker and singularity)
+        Clear all possibe container caches (ie docker and singularity)
```

Comment:
Can you move this into the tearDown instead of setUp ? That seems like more of a standard way of cleaning up resources that a test creates ?

Reply:
That surely makes sense for the singularity cache, which is created in a temp directory, but I'd keep it in `setUp()` for the `docker image rm` part, as we don't have control on what the docker system cache could contain in a user system.
I'll split this up in 2, if that's OK.

Reply:
(and yeah that's fine)